.github/workflows/release.yml

```diff
@@ -8,9 +8,10 @@ on:
 jobs:
   release:
     name: 'Release project'
-    uses: fingerprintjs/dx-team-toolkit/.github/workflows/release-sdk-changesets.yml@v1
+    uses: fingerprintjs/dx-team-toolkit/.github/workflows/release-sdk-changesets.yml@4b8be0c79af46ddb5b4812025fccb6177e1979ce
```

Comment:
Are we going to use SHA instead of tag to specify action versions, or is it only for testing purposes?